### PR TITLE
Add compress option

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by bcrowe** [comment]
:clap: Awesome!

**Comment by orendecor** [comment]
k88 you've made my day!
How come the change is only in 3 lines of code? Is the compress already implemented in the less engine?

**Comment by k88hudson** [comment]
@orendecor there is a compress option in less (http://lesscss.org/#using-less-configuration), although it's pretty simple (only removes whitespace). If you need more than that, you might need to use http://lesscss.org/usage/#v2-upgrade-guide-clean-css (I haven't included support for that in this PR since I don't need it, but it would be easy enough to do)

**Comment by orendecor** [comment]
What can be done in addition to removing whitespace?

**Comment by k88hudson** [comment]
@orendecor It will do things like merge adjacent rulesets or removing duplicate rules. Take a look here if you're interested :) https://github.com/jakubpawlowicz/clean-css#what-advanced-optimizations-are-applied

File: bin/autoless [hunk 1]
```diff
@@ -16,6 +16,7 @@ program
   .option('--no-notify', "Do not send any notifications")
   .option('--no-sound', "Do not play a sound with error notifications")
   .option('--source-map', "Generate source map files next to css files")
+  .option('--compress', "Compress output by removing some whitespaces")
```
Comment (jgonera) [inline bin/autoless]: Can we change this to "Compress output by removing whitespace"?

http://english.stackexchange.com/a/25371

**Comment by jgonera** [comment]
Thanks! Can you also update the readme?

File: bin/autoless (hunk 1 above)
Comment (k88hudson) [inline bin/autoless]: Sure, as long as you don't mind it being slightly different from the less cli/docs

**Comment by johanberonius** [comment]
👍

**Review by orendecor — APPROVED** [review]

**Comment by jgonera** [comment]
Sorry, I'm not going to maintain this project anymore since modern web development uses Webpack/Rollup for everything. I'm closing this PR before archiving the repo, feel free to fork it if you really need autoless.

Sorry for not merging the PR back in the day.

**Pull request closed without merging**